aspnet/whitepapers/side-by-side-with-10.md

```diff
@@ -75,17 +75,17 @@ The Aspnet\_regiis.exe provides two options for script mapping a Web application
 
 The path defines the Web application IIS metadata path, which is defined in the form of W3SVC/ROOT/{WebSiteNumber}/{Application\_Name}. For example, for a Web application called Portal located under the default Web site, the metabase path is W3SVC/1/ROOT/Portal.
 
-![](side-by-side-with-10/_static/image4.gif)
+![Screenshot that shows a Web application called Portal located under the default Web site, and the metabase path is W3SVC/1/ROOT/Portal.](side-by-side-with-10/_static/image4.gif)
 
 Note You can also use a tool called the Metabase Editor to get the metabase path. You can download this tool from the Microsoft Support site at [https://support.microsoft.com/default.aspx?scid=kb;en-us;232068.](https://support.microsoft.com/default.aspx?scid=kb;en-us;232068)
 
 - Run Aspnet\_regiis.exe -s W3SVC/1/ROOT/Portal to update the portal IIS script map and its subapplication.  
   
-    ![](side-by-side-with-10/_static/image5.gif)
+    ![Screenshot that shows to run Aspnet\_regiis.exe -s W3SVC/1/ROOT/Portal to update the portal IIS script map and its subapplication.](side-by-side-with-10/_static/image5.gif)
 
 - Run Aspnet\_regiis.exe -sn W3SVC/1/ROOT/Portal to update the portal IIS script map, without affecting applications in the portal?s subdirectories.  
   
-    ![](side-by-side-with-10/_static/image6.gif)
+    ![Screenshot that shows to Run Aspnet\_regiis.exe -sn W3SVC/1/ROOT/Portal to update the portal IIS script map, without affecting applications in the portal?s subdirectories.](side-by-side-with-10/_static/image6.gif)
```

Comment:
try writing out W3SVC/1/ROOT/Portal so it is easier for a screen reader to read out loud.

Also, portal?s is a typo.